projects/plugins/jetpack/tests/e2e/specs/blocks/free-blocks.test.js

```diff
@@ -25,147 +24,196 @@ test.describe.parallel( 'Free blocks', () => {
+				await blockEditor.viewPost();
+			} );
+
+			await test.step( 'Can assert that Subscribe block is rendered', async () => {
+				expect(
+					await block.isRenderedInFrontend( await PostFrontendPage.init( page ) ),
+					'Block should be displayed'
+				).toBeTruthy();
+			} );
 		} );
 	} );
 
-	test( 'Form block', async ( { page } ) => {
-		await test.step( 'Can visit the block editor and add a Form block', async () => {
-			const blockId = await blockEditor.insertBlock( FormBlock.name(), FormBlock.title() );
-
-			const block = new FormBlock( blockId, page );
-			await block.selectFormVariation();
-		} );
-
-		await test.step( 'Can publish a post with a Form block', async () => {
-			await blockEditor.selectPostTitle();
-			await blockEditor.publishPost();
-			await blockEditor.viewPost();
-		} );
-
-		await test.step( 'Can assert that Form block is rendered', async () => {
-			const frontend = await PostFrontendPage.init( page );
-			expect(
-				await frontend.isRenderedBlockPresent( FormBlock ),
-				'Block should be displayed'
-			).toBeTruthy();
-		} );
-	} );
-
-	test( 'Tiled Gallery block', async ( { page } ) => {
-		await test.step( 'Can visit the block editor and add a Tiled Gallery block', async () => {
-			const blockId = await blockEditor.insertBlock(
-				TiledGalleryBlock.name(),
-				TiledGalleryBlock.title()
-			);
-			const block = new TiledGalleryBlock( blockId, page );
-			await block.addImages();
-
-			await blockEditor.openSettingsSidebar();
-			await block.linkToAttachment();
-		} );
-
-		await test.step( 'Can publish a post with a Tiled Gallery block', async () => {
-			await blockEditor.selectPostTitle();
-			await blockEditor.publishPost();
-			await blockEditor.viewPost();
-		} );
-
-		await test.step( 'Can assert that Tiled Gallery block is rendered', async () => {
-			const frontend = await PostFrontendPage.init( page );
-			expect(
-				await frontend.isRenderedBlockPresent( TiledGalleryBlock ),
-				'Block should be displayed'
-			).toBeTruthy();
-		} );
-	} );
-
-	test( 'Subscribe block', async ( { page } ) => {
-		await prerequisitesBuilder( page ).withActiveModules( [ 'subscriptions' ] ).build();
-
-		await test.step( 'Can visit the block editor and add a Subscribe block', async () => {
-			const blockId = await blockEditor.insertBlock(
-				SubscribeBlock.name(),
-				SubscribeBlock.title()
-			);
-			const block = new SubscribeBlock( blockId, page );
-			await block.checkBlock();
-		} );
-
-		await test.step( 'Can publish a post with a Subscribe block', async () => {
-			await blockEditor.selectPostTitle();
-			await blockEditor.publishPost();
-			await blockEditor.viewPost();
-		} );
-
-		await test.step( 'Can assert that Subscribe block is rendered', async () => {
-			const frontend = await PostFrontendPage.init( page );
-			expect(
-				await frontend.isRenderedBlockPresent( SubscribeBlock ),
-				'Block should be displayed'
-			).toBeTruthy();
+	test.describe( 'Site editor', () => {
```

Comment:
I like that breakdown. We could easily move more of the blocks in there as well once this is merged. Nothing stopping us from running block tests in the 2 contexts now.